unified_planning/engines/mixins/compiler.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 AIPlan4EU project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unified_planning as up
+from enum import Enum, auto
+
+
+class CompilationKind(Enum):
+    GROUNDER = auto()
```

Comment:
Maybe `GROUNDING` instead of `GROUNDER` since this is a kind of compilation and not a tool?